Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/CqlTranslator.java

```diff
@@ -230,7 +230,7 @@ public static String convertToJSON(Library library) throws JAXBException {
         return writer.getBuffer().toString();
     }
 
-    private static void loadModelInfo(File modelInfoXML) {
+    public static void loadModelInfo(File modelInfoXML) {
```

Comment:
I think this is a leftover from a couple of years ago from another project.  I was creating javascript models on the fly in a UI and needed access to the method for things to work correctly, at least thats as best as I can remember.  I can take this out if needed.